README.md

```diff
@@ -100,7 +100,7 @@ document.JsonObjectSize > Constants.COMPRESSED_DATA_MAX_SIZE_IN_BYTES
 ```
 
 ### Pushing a document with large binary data size
-The Push API has a hard limit of 5MB (compressed) on the document's binary data. When the size exceeds that value you typically need to request an upload URI to an S3 Bucket. You would then put your document binary data in that bucket and refer to it by ID when pushing the target document with the Push API. However, lucky you, the SDK handles this automatically. Thus, the SDK compresses the content, and if it exceeds 5MB (compressed), it will do the needed logic. Take note that the maximum size of a document is 256MB (compressed). 
+The Push API has a hard limit of 5MB (compressed) on the document's binary data. When the size exceeds that value you typically need to request an upload URI to an S3 Bucket. You would then put your document binary data in that bucket and refer to it by ID when pushing the target document with the Push API. However, lucky you, the SDK handles this automatically. Thus, the SDK compresses the content, and if it exceeds 5MB (compressed), it will do the needed logic. Take note that the maximum size of a document is 256MB (compressed).
```

Comment:
```suggestion
The Push API has a hard limit of 5MB (compressed) on the document's binary data. When the size exceeds that value, you typically need to request an upload URI to an S3 Bucket. You would then put your document binary data in that bucket and refer to it by ID when pushing the target document with the Push API. However, lucky you, the SDK handles this automatically. Thus, the SDK compresses the content, and if it exceeds 5MB (compressed), it will do the needed logic. Take note that the maximum size of a document is 256MB (compressed).
```

Reply:
Here: "You would then put your document binary data in that bucket and refer to it by ID when pushing the target document with the Push API"

I suppose we refer to the bucket's ID? If so, I'd be a bit more specific. It's a bit confusing whether we are referring to the binary data's ID or the bucket's ID.

Reply:
@ClaudineL I think you made this change. Would you be able to confirm :)

Reply:
The only change I made was a space at the end of the sentence that was deleted 🙂 But if you think the original doc is too confusing I'll clarify it.

Reply:
Looking at the code, the ID actually refers to the fileId returned by the request to get a file container, so I'm not sure if referring to it as a bucket is accurate, it's a separate kind of data structure.